src/components/heading/heading.js

```diff
@@ -25,6 +31,9 @@ const Heading = ({ title, children, isAscending, setIsAscending }) => {
               }}
             ></p>
           </span>
+          {isShowAuthor && (
+            <span className="hidden lg:inline absolute right-28">Added BY</span>
```

Comment:
Think this should be, "Updated by"

```suggestion
            <span className="hidden lg:inline absolute right-28">Updated by</span>
```

Reply:
I have talked about this with @bradystroud . Since it shows the person who created the rule, I think it's better to use 'Added By' here."

Reply:
Ah yes, just re-read the issue. How confusing 😅😅